buildkite/graphql/node.graphql

```diff
@@ -5,6 +5,9 @@ query getNode(
         ... on Pipeline {
             ...PipelineValues
         }
+        ... on PipelineTemplate {
+            ...PipelineTemplateValues
+        }
         ... on TeamMember {
             ...TeamMemberValues
         }
```

Comment:
Consistency on `fields` or `values` might be worth considering.

Reply:
Im for the former - I can head the renaming of the other two Fragments in another PR :)